docs/stats-engine/variance.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Variance 
```

Comment:
Should we call this section Standard error?  It's the term we use in console, and would help distinguish variance of the metric from the variance of the mean.